charts/rancher-monitoring/v0.1.5/questions.yml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.4.9-rc1
```

Comment:
Same comment as within 2.4 check.

Shouldn't this be 2.4.14-rc1 and there be a max version of the monitoring version v0.1.4, so there is only 1 monitoring chart for k8s 1.16?

Reply:
https://github.com/rancher/system-charts/pull/423

Reply:
Also, to clarify based on offline discussion, there will still be 2 charts for Monitoring in k8s 1.16 even after the above PR is resolved: 0.1.5 and 0.2.1

However, currently 0.1.4 and 0.2.0 are also visible in 2.5.6 for k8s 1.16 users, which is what the above PR resolves by adding a rancher_max_version to those charts.